plugins/commands/evaluator/src/main/kotlin/EvaluatorCommand.kt

```diff
@@ -273,19 +274,28 @@ class EvaluatorCommand : OrtCommand(
             ortResultInput = ortResultInput.addPackageCurations(packageCurationProviders)
         }
 
-        val packageConfigurationProvider = if (ortConfig.enableRepositoryPackageConfigurations) {
-            CompositePackageConfigurationProvider(
-                SimplePackageConfigurationProvider(ortResultInput.repository.config.packageConfigurations),
-                DirectoryPackageConfigurationProvider(packageConfigurationsDir)
-            )
-        } else {
-            if (ortResultInput.repository.config.packageConfigurations.isNotEmpty()) {
-                logger.info { "Local package configurations were not applied because the feature is not enabled." }
+        val enabledPackageConfigurationProviders = buildList {
+            val repositoryPackageConfigurations = ortResultInput.repository.config.packageConfigurations
+
+            if (ortConfig.enableRepositoryPackageConfigurations) {
+                add(SimplePackageConfigurationProvider(repositoryPackageConfigurations))
+            } else {
+                if (repositoryPackageConfigurations.isNotEmpty()) {
+                    logger.info { "Local package configurations were not applied because the feature is not enabled." }
+                }
             }
 
-            DirectoryPackageConfigurationProvider(packageConfigurationsDir)
+            val providerFromOption = DirectoryPackageConfigurationProvider(packageConfigurationsDir)
+            add(providerFromOption)
+
+            val packageConfigurationProviders =
+                PackageConfigurationProviderFactory.create(ortConfig.packageConfigurationProviders)
+            addAll(packageConfigurationProviders.map { it.second })
         }
 
+        val packageConfigurationProvider =
+            CompositePackageConfigurationProvider(*enabledPackageConfigurationProviders.toTypedArray())
```

Comment:
For `PackageCurations` we had the requirement to associate the package configurations with the ID of the provider (in order to re-apply the curations for certain providers only).
Would that requirement also be here in an analog way?

I'm asking that because for `PackageCurations` the `Composite*Provider` had to be eliminated since it prevented exactly that.

Reply:
> in order to re-apply the curations for certain providers only

I'm not aware of that functionality. Why is it required for curations?
I'd need to understand that to be able to say if it would also be necessary for package-configurations.